src/main/kotlin/javabot/operations/URLTitleOperation.kt

```diff
@@ -44,8 +49,8 @@ class URLTitleOperation @Inject constructor(bot: Javabot, adminDao: AdminDao,
         val title = if (titlesToPost.size == 1) "title" else "titles"
         responses.add(Message(event, "${event.user.nick}'s $title: " +
                 titlesToPost.joinToString(" | ", transform = { s ->
-                    if (s.length > 250) {
-                        "\"${s.substring(0, 260)}...}\""
+                    if (s.length > 240) {
+                        "\"${s.substring(0, 240)}...\"\""
```

Comment:
maybe use """ so you don't need to escape

Reply:
That's uuuuugly but doable (and done)